src/release-pr.ts

```diff
@@ -88,60 +70,65 @@ export interface OpenPROptions {
 }
 
 export class ReleasePR {
-  apiUrl: string;
-  defaultBranch?: string;
   labels: string[];
   fork: boolean;
   gh: GitHub;
   bumpMinorPreMajor?: boolean;
-  repoUrl: string;
-  token: string | undefined;
   path?: string;
   packageName: string;
   monorepoTags: boolean;
   releaseAs?: string;
   snapshot?: boolean;
   lastPackageVersion?: string;
-  changelogSections?: [];
-  releaseType: string;
-  // Prefix for tags/branches
-  packagePrefix: string;
+  changelogSections?: ChangelogSection[];
 
-  constructor(options: ReleasePROptions) {
+  constructor(options: ReleasePRConstructorOptions) {
     this.bumpMinorPreMajor = options.bumpMinorPreMajor || false;
-    this.defaultBranch = options.defaultBranch;
     this.fork = !!options.fork;
     this.labels = options.label
```

Comment:
Feels a bit weird that `label` starts as a string, that's split, and then we pass it around as an array of strings to the GitHub instance.

We could coerce labels into an array in the CLI, using `.coerce` and in the GitHub action, and perhaps make labels an array from the factory onwards?

Reply:
I like it. Do you want me to do that in this PR?

Reply:
I'm doing it in this PR - and it's leading to another `GitHubRelease` option culling: that class doesn't need `this.labels` now that it has a `releasePR` instance. I hadn't looked closely but it turns out ReleasePR.labels is supposed be the same as GitHubRelease.labels. Back before GitHubRelease had a releaser instance that duplication was necessary, but not anymore